### ISSUE TITLE
batch_size influence on the performance

### ISSUE THREAD

**Issue by bspmpa** [Event-AHU/OpenPAR#13]
Hi, when reproducing the performance of the PA100k of your work, I try to use 24, 32, 64 as the batch size. But the performance is lower than the ones with 16 as the batch size. I am wondering can you shed some light on this issue. I assume for large dataset such as PA100k, a larger batch size would be benefical.

**Comment by 1125178969** [comment]
In this framework, the prompt tokens are sensitive to the learning rate, so I think you may not have adjusted the learning rate for the prompt token while modifying the batch_size.

**Comment by 1125178969** [comment]
And the relationship between batch size and precision is not positively correlated, we experimented on most of the dataset is also bacth size of 16 when the best results

**Comment by bspmpa** [comment]
Thanks for your reply. I have adjusted the clip_lr, from 4e-3 to 8e-3. The acc is only around 79% (vs 83% in paper). I agree with your experiment findings on batchsize, although this is a little bit baffling. Is it because we are fine-tuning the large model so smaller batchsize is better?

**Comment by 1125178969** [comment]
Understandably, the learning process of prompt seems to be related to the bacth size, and higher batch size may be harmful to its learning, but we have not run any experiments on this.

**Comment by 1125178969** [comment]
But it should be possible to reproduce the results in the article on a bacth size of 16

**Comment by bspmpa** [comment]
yes, the results are close but has 0.5% gap. that is why I want to do some parameter fine-tuning.
